x-pack/plugins/security_solution/server/lib/detection_engine/rule_schema/model/rule_schemas.ts

```diff
@@ -87,11 +88,18 @@ import { ResponseActionRuleParamsOrUndefined } from '../../../../../common/detec
 const nonEqlLanguages = t.keyof({ kuery: null, lucene: null });
 
 export type AlertSuppressionCamel = t.TypeOf<typeof AlertSuppressionCamel>;
-const AlertSuppressionCamel = t.exact(
-  t.type({
-    groupBy: AlertSuppressionGroupBy,
-  })
-);
+const AlertSuppressionCamel = t.intersection([
+  t.exact(
+    t.type({
+      groupBy: AlertSuppressionGroupBy,
+    })
+  ),
+  t.exact(
+    t.partial({
+      duration: AlertSuppressionDuration,
+    })
+  ),
+]);
```

Comment:
Should this 🐪 be over in the `securitysolution-io-ts-alerting-types` package, like the `RuleActionCamel`?

https://github.com/elastic/kibana/blob/21de750b3cff4d4ad27d07c77135327e644b3bee/packages/kbn-securitysolution-io-ts-alerting-types/src/actions/index.ts#L41-L48

Reply:
Yeah I don't think it should be here in `rule_schemas.ts`, not sure about the package though. I moved it to `query_attributes.ts` for now to live with the rest of the alert suppression schemas. If the packages are preferred for the schemas we may want to move more of the schemas over?

Reply:
Yeah, it might be time to loop around again and see which remaining schemas can find a new home over in packages.